akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -40,7 +40,7 @@
 
         <!-- These are dependent on runtime environment and cannot be customized by users -->
         <maven.compiler.release>21</maven.compiler.release>
-        <kalix-runtime.version>1.3.5-bc0233b</kalix-runtime.version>
+        <kalix-runtime.version>1.3.6</kalix-runtime.version>
```

Comment:
why did we have a non final version here?

Reply:
This is against the feature branch, which depended on unreleased SPI changes, was there to make it build